### DISCUSSION TITLE
regolith-session-flashback appears to conflict with regolith-look packages

### DISCUSSION THREAD

**Discussion by scmbradley** [regolith-linux/regolith-desktop#826] [Q&A]
Here's what happens whenever I try to install a regolith-look package:

```
$ sudo apt install regolith-look-nord
[sudo] password for seamus: 
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies.
 regolith-session-flashback : Conflicts: regolith-styles but 2.9.7-1 is to be installed
E: Error, pkgProblemResolver::Resolve generated breaks, this may be caused by held packages.
```
I haven't tried them all, but ayu-dark, solarized-dark and nord all have the same issue. What could be causing this?

**Comment by JupiLogy** [comment]
it's been a while since this was posted but I'm having exactly the same issue with ubuntu 24.10. After some issues (which turned out to be from drivers) I tried completely uninstalling and reinstalling my regolith packages.

Preemptively confirming that I'm using the recommended ppa, and have no software held back.

The conflict is between `regolith-styles` and `regolith-session-flashback`, 
and also (now) between `regolith-styles` and `regolith-session-sway`.

Installing styles first means it gets removed when trying to install the session. Installing the session first means that also gets removed when trying to install the styles. I'm willing to test further if anyone has any ideas.

_~skips away to play with my neglected gentoo in the meantime_
